tools/certa/package.json

```diff
@@ -31,13 +31,13 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
+    "playwright": "~1.35.1",
```

Comment:
@ben-polinsky correct me if im wrong, but we went with playwright/test because it didnt install more browser drivers than we needed

Reply:
Yes correct

Reply:
Imo it isn’t too much overhead cost for the security benefit, specially since this is a public repo. Thoughts?

I was thinking maybe we use same approach doe auth-clients repo too instead of the install js script approach?

Reply:
when we update the auth-clients pkg to pull in this, lets update the install step like this